Source/Core/VideoCommon/AbstractTexture.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+#include "Common/MathUtil.h"
+#include "VideoCommon/TextureConfig.h"
+#include "VideoCommon/VideoCommon.h"
+
```

Comment:
A simple search shows 14 files, some probably overlapping with other files in this PR but that's still 14 files for a change that really has little to do with this concept.  Unless you feel strongly about this, I'd rather save this cleanup for another PR.